data-prepper-plugins/geoip-prepper/src/test/java/com/amazon/dataprepper/plugins/prepper/geoip/provider/LocationDataTest.java

```diff
@@ -0,0 +1,34 @@
+package com.amazon.dataprepper.plugins.prepper.geoip.provider;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class LocationDataTest {
+    private static final String TEST_TOSTRING_OUTPUT = "LocationData{countryName='a', subdivisionName='b', cityName='c'}";
+    private static LocationData location;
+
+    @BeforeEach
+    public void setup() {
+        location = new LocationData("a", "b", "c");
+
+    }
+
+    @Test
+    public void testToString() {
+        Assertions.assertEquals(TEST_TOSTRING_OUTPUT, location.toString());
+    }
+
+    @Test
+    public void testEquals() {
+        LocationData location2 = new LocationData("a", "b", "c");
+        LocationData location3 = new LocationData("a", "b", "f");
```

Comment:
nit - name something like `expectedLocation` and `unexpectedLocation` so we don't have to guess what `2` and `3` represent.

Though tbh we could probably skip testing this class if it's going to remain just a simple struct